routes/actions.js

```diff
@@ -210,7 +213,7 @@ router.get('/register', function(req, res) {
 });
 
 router.get('/register/:code', function(req, res, next) {
-  const { code } = req.params;
+  const { code } = req.params.code;
```

Comment:
I'm assuming this is an accidental/unintentional change. It breaks the invite link logic: the original assignment is a destructuring assignment; your change will always result in `undefined`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring

Reply:
semi-unintentional,  you tend to mix params and query when talking about scope so I changed this early on thinking I would need to differentiate params.